velox/functions/sparksql/aggregates/CollectSetAggregate.cpp

```diff
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#include <type/Type.h>
```

Comment:
I think this line is unnecessary. Maybe remove it.

Reply:
yes, removed